### DISCUSSION TITLE
Github Account Got Suspended Without Any Reason

### DISCUSSION THREAD

**Discussion by paradox714** [community/community#134410] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was writing a discussion post on student developer pack 10 anniversary and then after submitting when i started working on my project i got logged out , I have student developer pack integrated with that account and all my source codes , accessbility to tools and services like aws , heroku etc are gone with that. I was Working on so many projects and open source contribution was about to be made.

Github hadn't send me any valid reason email for suspension of account , neither they respond . 

That was my main professional account , is therr any way to get it faster????

**Comment by SAHELISAHAA** [comment]
@paradox714 same thing happened with me

**Reply by DEXTER12n9** [reply]
Today on 2nd aug??

**Reply by SAHELISAHAA** [reply]
Yes

**Reply by SAHELISAHAA** [reply]
@DEXTER12n9 same thing happened with you?

**Reply by all6h** [reply]
Do you received any mail from github?

**Reply by DEXTER12n9** [reply]
@SAHELISAHAA yes

**Reply by SAHELISAHAA** [reply]
I raised a ticket

**Reply by DEXTER12n9** [reply]
@rexsql A mail to reverify my student dev pack within 30 days to enjoy within disruption that's it ,i think they have to send some kind of mail and since suspension cause the stopage of sdp that's why though

**Reply by SAHELISAHAA** [reply]
@DEXTER12n9 I got the same mail

**Reply by DEXTER12n9** [reply]
> @DEXTER12n9 I got the same mail

I just hope they give our accounts back as it is asap can't wait for weeks and months .

**Reply by SAHELISAHAA** [reply]
@DEXTER12n9 same bro

**Reply by SAHELISAHAA** [reply]
@vincentchanme did you raised a ticket ?

**Reply by SAHELISAHAA** [reply]
@vincentchanme some people got their account within a week

**Reply by DEXTER12n9** [reply]
> same to me, already weeks for waiting.

😱 Have they mentioned Any reason for yours?

**Reply by DEXTER12n9** [reply]
> @vincentchanme did you raised a ticket ?

Yeah I have done all formalities

**Reply by SAHELISAHAA** [reply]
@vincentchanme I also checked it . Don't know what gonna happen , All my hardwork will go down in vain , I will try to connect with them through social media let's see what happen

**Reply by DEXTER12n9** [reply]
> @vincentchanme I also checked it . Don't know what gonna happen , All my hardwork will go down in vain , I will try to connect with them through social media let's see what happen

I have already done this through LinkedIn , Instagram whoever possible

**Comment by SAHELISAHAA** [comment]
@DEXTER12n9 when did your account got suspended ?

**Reply by DEXTER12n9** [reply]
> @DEXTER12n9 when did your account got suspended ?

2nd aug

**Reply by MohammadRafique12** [reply]
@DEXTER12n9 bro can you please tell me how yours account was back ?? please www.linkedin.com/in/mohammad-rafique1 this is my lindedin contact me

**Reply by MohammadRafique12** [reply]
my account got also suspend @DEXTER12n9

**Reply by MohammadRafique12** [reply]
i am trying for a month to resolve this @DEXTER12n9

**Comment by ghost** [comment]
I have been facing this issue for more than 24 hours now, and I am fuming due to the absolutely inhumane and unprofessional behavior of the team in suspending my account without any notice whatsoever.

I am also unable to access my organizational account, as I was the sole owner. 

Even government agencies provide reasonable notice to respond, but this situation feels like a dictatorship and an abuse of power. I had a lot of trust in GitHub, but with such actions, I have no choice but to consider diversifying my work with other vendors once my organizational account is reinstated and my repositories are restored locally. 😡

**Reply by SAHELISAHAA** [reply]
true

**Reply by ashish5kmax** [reply]
same here

**Comment by ghost** [comment]
@paradox714: While raising support ticket, GitHub Copilot informed me that only one free personal account per person is allowed. According to GitHub’s terms and conditions, if we manage multiple organizations, we should link them all to that one free personal account rather than creating a new personal account for each organization.

Now that we are aware of this policy, GitHub could at least reinstate the account so that we have 24 hours to address the issue by merging or deleting accounts as appropriate. Waiting for their response.

**Comment by SAHELISAHAA** [comment]
@prnam same bro , I didn't got any email about it , and now during the middle of my work my account got suspended it . It will hamper my carrier a lot .

**Comment by LiteBrite82** [comment]
Hi @paradox714 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by ghost** [reply]
It’s been close to 48 hours, and no one has responded to my support ticket, @LiteBrite82. It’s clearly a hostile takeover of our IP, and I regret trusting this platform to handle it professionally. We need to advocate for strong enforcement of local data regulations and processes to prevent such outrageous behavior from large companies dictating rules globally.

**Reply by DEXTER12n9** [reply]
> Hi @paradox714 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

I have done all things , yet no help

**Reply by SAHELISAHAA** [reply]
Same here

**Reply by ghost** [reply]
The only way left is to send legal notice. Tedious and pricey process. I would wait for a week to see if they would respond. 

https://docs.github.com/en/site-policy/other-site-policies/guidelines-for-legal-requests-of-user-data#submitting-requests

Regardless of the outcome, the Indian government needs to know about this, and strict measures must be taken to protect our IP and protect national interests.

**Reply by Sahelisaha04** [reply]
@DEXTER12n9 any update on your account?

**Reply by DEXTER12n9** [reply]
> @DEXTER12n9 any update on your account?

No

**Reply by ghost** [reply]
Here is the update I got for my reinstate request. FYR.

<img width="826" alt="image" src="https://github.com/user-attachments/assets/02a511dc-a218-4dde-b01b-40f0f2df22d1">

**Reply by ghost** [reply]
@vincentchanme - Nothing changed since my last message here in this thread. I have also raised request to convert my organistion as GitLab Enterprise but no luck. Raised appeal to reinstate Organization but no luck.

I have moved out rest of my work to GitLab for now and building my own remote Git service, and if everything goes well one day it will be one of the major remote Git provider and grab market share of GitHub. 😡 ⚔️

**Comment by Sahelisaha04** [comment]
Let me know anyone got their account back or not

**Reply by ghost** [reply]
I have requested for reinstatement of my don't and said in it that I will merge it all into 1 account and all the repos and organisation, but they have not responded back.

**Comment by alanzimm85** [comment]
Please help, i need to get my account back, i sent a lot of curriculums with projects on this github account. I didn´t do nothing wrong. It's not from this account, i can´t acces the other. Ticket ID: 2921645

You have been suspended from GitHub.
 
This moderation action was taken after we received a report and determined that your account violates our Acceptable Use Policy prohibiting the use of our platform in direct support of unlawful attacks that cause technical harms. We do not allow anyone to use our platform in direct support of unlawful attacks that cause technical harms, such as using GitHub as a means to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Technical harms means overconsumption of resources, physical damage, downtime, denial of service, or data loss, with no implicit or explicit dual-use purpose prior to the abuse occurring.

When making content moderation decisions, we consider information from a variety of sources, including: account profile data, information contained in submitted reports/notices or discovered through our own voluntarily initiated investigations, and context around the contents of the repository.

If you wish to regain access or would like to dispute that a violation occurred and can provide additional information to show that a different decision should have been reached, please review our [Appeal and Reinstatement Policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement) and submit a request via our [form](https://support.github.com/contact/reinstatement).

**Comment by DEXTER12n9** [comment, marked as answer]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> I was writing a discussion post on student developer pack 10 anniversary and then after submitting when i started working on my project i got logged out , I have student developer pack integrated with that account and all my source codes , accessbility to tools and services like aws , heroku etc are gone with that. I was Working on so many projects and open source contribution was about to be made.
> 
> Github hadn't send me any valid reason email for suspension of account , neither they respond . 
> 
> That was my main professional account , is therr any way to get it faster????

Hey Everyone I got my GitHub account back , i started this Thread As I was frustrated at that moment but support staff is very active and they have reinstated my account within 6 days as orginal. So I'd say have faith in support team 😅

**Reply by Sahelisaha04** [reply]
@DEXTER12n9  What was your ticket number?

**Reply by Sahelisaha04** [reply]
@vincentchanme any updates?

**Reply by ashish5kmax** [reply]
even my primary account got hacked and suspended can anyone help me out the account name was @ashish4kmax please help me out it was my main professional account and as companies are coming I need it very much for my projects please.

**Reply by ghost** [reply]
I have not got any postive response. Here is what I received from them. FYR.

<img width="826" alt="image" src="https://github.com/user-attachments/assets/6b55e1cf-cb6c-4a4d-93a4-f4e23e94ba4d">

**Reply by ashish5kmax** [reply]
feeling really bad for you @prnam how these hackers are so easily use our GitHub bypassing security.

**Reply by ghost** [reply]
It is not hackers. It was GitHub security bots automatically blocked based on my actions. I just created seperate user accounts for all orgs I was working on, and they just refuse access nor they allowing me to revert my actions of all orgs to 1 account (this account). They are also not releasing even when I said I am ready to pay. Lost all my Intellectual Property. Have to fight in court now and central government has to send notice to US government and then it take effect. Lot of unnecessary issue or process they are creating when I am ready pay for accounts.

**Reply by ashish5kmax** [reply]
ohh

**Reply by ghost** [reply]
All I can say after this incident is India needs its own remote Git platform service. Cannot depend on outsiders who dictate like this and loot our Intellectual Property. 

Very very dangeours given our country is growing really fast with lot of great technologies and also many students build great innovative product while blindly trusting that GitHub is safe and won't harm them.

**Reply by ashish5kmax** [reply]
yes

**Comment by ashish5kmax** [comment]
even my primary account got hacked and suspended can anyone help me out the account name was @ashish4kmax please help me out it was my main professional account and as companies are coming I need it very much for my projects please.

**Comment by Schizok** [comment]
@prnam hello bro, my account also got smothing like this i mean i dont know if its suspended or anything else but i cant login or see my projects please tell how you got your account back
